data_and_models/pipelines/sentence_embedding/dvc.yaml

```
@@ -1,4 +1,38 @@
 stages:
+  training_transformers:
+    cmd: >-
+    python training_transformers/train.py
+    --model_name_or_path clagator/biobert_v1.1_pubmed_nli_sts
+    --train_data_file training_transformers/sentences-filtered_11-527-877.txt
+    --eval_data_file training_transformers/biosses_sentences.txt
+    --line_by_line
+    --mlm
+    --output_dir ../../models/sentence_embedding/biobert_cord19_v1/
```

```suggestion
    --output_dir ../../models/language_modeling/biobert_cord19_v1/
```

The pre-trained model is actually already tracked by DVC in `models/language_modeling/`.

---

Done in 2bd998f.